app/models/alchemy/picture.rb

```diff
@@ -132,6 +133,19 @@ def url_class=(klass)
         @_url_class = klass
       end
 
+      def alchemy_resource_filters
+        [
+          {
+            name: :filter,
```

Comment:
Ah, that's why we have the `params[:filter][:filter]` issue below? What about renaming this to 

```suggestion
            name: :scope,
```

That way it is also very clear what this resource filter type uses under the hood. WDYT?

Reply:
Absolutely!

Reply:
I have an issue with the name `scope` because every filter defined in `alchemy_resource_filters` is a scope, not just this one here. The reason why its hard to find a name for this is because its a mix of scopes, all unrelated to each other. This brings me to the name `misc` (abbr of miscellaneous) as it describes exactly that.

Reply:
I like `misc`